### PR TITLE
fix(python sdk): adds namespace config

### LINKED ISSUE #279: feat(python sdk): namespace config var
[typescript/#27](https://github.com/hatchet-dev/hatchet-typescript/pull/27) added a namespace configuration prop to improve the local developer experience for multiple devs on a single hatchet tenant.

We need the same namespacing functionality for the Python SDK 

Related #278

### PR DESCRIPTION
**Pull request by aeswibon**
# Description

- Adds new namespace config to `python-sdk`  
- Fixes #279

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## What's Changed

- [ ] Add a list of tasks or features here...

### CODE REVIEW

File: python-sdk/hatchet_sdk/clients/events.py [hunk 1]
```diff
@@ -1,37 +1,44 @@
...
+        self.namespace = namespace
 
     def push(self, event_key, payload):
         try:
-            payload_bytes = json.dumps(payload).encode('utf-8')
+            payload_bytes = json.dumps(payload).encode("utf-8")
         except json.UnicodeEncodeError as e:
             raise ValueError(f"Error encoding payload: {e}")
 
+        event_key = f"{self.namespace}:{event_key}"
```
Comment (abelanger5) [inline python-sdk/hatchet_sdk/clients/events.py:41]: This doesn't seem to match the implementation of the TS SDK: https://github.com/hatchet-dev/hatchet-typescript/blob/eaca1fcfd416665467a6fa8bcc0ac0989dd7512f/src/clients/event/event-client.ts#L33.

File: python-sdk/hatchet_sdk/worker.py [hunk 2]
```diff
@@ -2,49 +2,75 @@
...
 class Worker:
-    def __init__(self, name: str, max_runs: int | None = None, debug=False, handle_kill=True):
+    def __init__(
+        self,
+        name: str,
+        max_runs: int | None = None,
+        debug=False,
+        handle_kill=True,
+    ):
         self.name = name
```
Comment (abelanger5) [inline python-sdk/hatchet_sdk/worker.py:46]: We should be modifying the `self.name` to be namespaced as well, to match the TS SDK: https://github.com/hatchet-dev/hatchet-typescript/pull/27/files#diff-2bba1bd36b8922955278a73eeaca7a34816e0ea0a63c498f0f6cfc028010f40d

Comment (abelanger5) [reply python-sdk/hatchet_sdk/worker.py:46]: Note that the `event_triggers` when creating the workflow should also be modified.

**Review by abelanger5 — CHANGES_REQUESTED** [review]
Hey @aeswibon thanks for the PR! There are a few things which don't match up with the typescript SDK that I flagged. 

It might be easier if you make a separate PR for Python formatting and format all `.py` files via that PR. We can then merge that in ahead of this PR so it's easier to see what's changed.

**Comment by aeswibon** [comment]
Sure, I will open a PR to format all the `.py` files, then fixed all the changes suggested.

**Comment by grutt** [comment]
tracking in #354

**Pull request closed without merging**